src/reynir/Greynir.grammar

```diff
@@ -1187,8 +1197,10 @@ Sem → "sem:st"
 $score(+10) VirðistSem # Sennilega rétt greint
 
 SögnFinnstBotn →
-     'vera:so'_1_nf_nh NlSagnfylling_nf                 # Finnst þeim vera slagsíða í umfjölluninni
-     | NlFrumlagÞað? LoSamanb_et_nf_hk AðLiður?
+    'vera:so'_1_nf_nh NlSagnfylling_nf                 # Finnst þeim vera slagsíða í umfjölluninni
+    | NlFrumlagÞað? LoSamanb_et_nf_hk AðLiður?
+    | NhRuna
+    | so_expl_op_3p_et
```

Comment:
Would be good to add comments with examples here

Reply:
Done